dali/operators/math/expressions/arithmetic_meta.h

```diff
@@ -367,6 +378,70 @@ REGISTER_BINARY_IMPL(ArithmeticOp::add, +);
 REGISTER_BINARY_IMPL(ArithmeticOp::sub, -);
 REGISTER_BINARY_IMPL(ArithmeticOp::mul, *);
 
+template <typename Backend>
+struct arithm_meta<ArithmeticOp::min, Backend> {
+  template <typename L, typename R>
+  using result_t = binary_result_t<L, R>;
+
+  template <typename L, typename R>
+  DALI_HOST_DEV static constexpr result_t<L, R> impl(L l, R r) {
+    auto l_ = static_cast<result_t<L, R>>(l);
+    auto r_ = static_cast<result_t<L, R>>(r);
+    return l_ <= r_ ? l_ : r_;
+  }
+
+  static inline std::string to_string() {
+    return "min";
+  }
+
+  static constexpr int num_inputs = 2;
+  static constexpr int num_outputs = 1;
+};
+
+template <typename Backend>
+struct arithm_meta<ArithmeticOp::max, Backend> {
+  template <typename L, typename R>
+  using result_t = binary_result_t<L, R>;
+
+  template <typename L, typename R>
+  DALI_HOST_DEV static constexpr result_t<L, R> impl(L l, R r) {
+    auto l_ = static_cast<result_t<L, R>>(l);
+    auto r_ = static_cast<result_t<L, R>>(r);
+    return l_ >= r_ ? l_ : r_;
+  }
+
+  static inline std::string to_string() {
+    return "max";
+  }
+
+  static constexpr int num_inputs = 2;
+  static constexpr int num_outputs = 1;
+};
+
+
+template <typename Backend>
+struct arithm_meta<ArithmeticOp::clamp, Backend> {
+  template <typename T, typename Min, typename Max>
+  using result_t = binary_result_t<binary_result_t<T, Min>, Max>;
+
+  template <typename T, typename Min, typename Max>
+  DALI_HOST_DEV static constexpr result_t<T, Min, Max> impl(T v, Min lo, Max hi) {
+    auto v_ = static_cast<result_t<T, Min, Max>>(v);
+    auto lo_ = static_cast<result_t<T, Min, Max>>(lo);
+    auto hi_ = static_cast<result_t<T, Min, Max>>(hi);
+    auto lo_clamp_ = v_ <= lo_ ? lo_ : v_;
+    return lo_clamp_ >= hi_ ? hi_ : lo_clamp_;
```

Comment:
```suggestion
    return clamp(v_, lo_, hi_);
```
from `dali/core/math_util.h`